site/content/docs/02-template-syntax/02-markup-elements.md

```diff
@@ -0,0 +1,134 @@
+---
+title: Markup elements
```

Comment:
Maybe its just me, but it doesn't really explain what I'm gonna get in this page.

Maybe `Basics`? `Template Basics`?